consultation_analyser/consultations/jinja2/privacy.html

```diff
@@ -10,53 +10,60 @@ <h1 class="govuk-heading-l">Privacy notice for Consultation analyser</h1>
 
     <h2 class="govuk-heading-m">Your data</h2>
     <h3 class="govuk-heading-s">Purpose</h3>
-    <p class="govuk-body">The purpose(s) for which we are processing your personal data are:</p>
+    <p class="govuk-body">The purpose for which we are processing your personal data are:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>enabling login to the platform</li>
-      <li>(with consent) contacting to support evaluation</li>
+      <li>To be used in email correspondence as part of the feedback and review process.</li>
+      <li>Credentials for access to Cabinet Office provided i.AI Consultation analyser platform.</li>
     </ul>
 
     <h3 class="govuk-heading-s">The data</h3>
     <p class="govuk-body">We will process the following personal data:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>organisational email address</li>
-      <li>department</li>
-      <li>profession/area of work</li>
+      <li>Name (via email)</li>
+      <li>Work Email</li>
+      <li>Department</li>
+      <li>Profession</li>
     </ul>
 
-
-    <h3 class="govuk-heading-s">Legal basis of processing</h3>
+    <h3 class="govuk-heading-s">Legal basis for processing</h3>
     <p class="govuk-body">The legal basis for processing your personal data is:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>data subject consents</li>
+      <li>It is necessary for the purposes of the legitimate interests pursued by the controller or by a third party. In this case those are providing secure access to Cabinet Office platforms and applications. As well as enabling you to participate in electronic communications as part of the feedback on the process.</li>
     </ul>
 
     <h3 class="govuk-heading-s">Recipients</h3>
-    <p class="govuk-body">Your personal data will be shared by us with analysts/researchers within the Cabinet Office if you have consented to be contacted further.</p>
-    <p class="govuk-body">As your personal data will be stored on our IT infrastructure it will also be shared with our data processors who provide email, and document management and storage services.</p>
+    <p class="govuk-body">Your personal data will be shared by us with:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>For platform access it will be provided to the provider of the platform.</li>
```

Comment:
Style q: should we remove all full stops from the end of `<li>`s?

Reply:
Good question. And the answer to said question is:
https://design-system.service.gov.uk/styles/lists/#bulleted-lists
I will update all list items on the page.